pkg/rescheduler/strategies/duplicates.go

```diff
@@ -21,49 +21,60 @@ import (
 	"strings"
 
 	"k8s.io/kubernetes/pkg/api/v1"
+	//TODO: Change to client-go instead of generated clientset.
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 
 	"github.com/aveshagarwal/rescheduler/pkg/rescheduler/evictions"
 	podutil "github.com/aveshagarwal/rescheduler/pkg/rescheduler/pod"
 )
 
+// This strategy evicts all duplicate pods on node. A pod is said to be a duplicate of other if
+// both of them have same name, kind and are within the same namespace. As of now, this strategy won't
+// evict daemonsets, mirror pods and pods with local storages.
+
 //type creator string
-type DuplicatePodsMap map[string][]*v1.Pod
+type duplicatePodsMap map[string][]*v1.Pod
 
-func RemoveDuplicatePods(client clientset.Interface, policyGroupVersion string, nodes []*v1.Node) error {
+// RemoveDuplicatePods removes the duplicate pods on node.
+func RemoveDuplicatePods(client clientset.Interface, policyGroupVersion string, nodes []*v1.Node) (int, error) {
+	podsEvicted := 0
 	for _, node := range nodes {
 		fmt.Printf("\nProcessing node: %#v\n", node.Name)
-		dpm := RemoveDuplicatePodsOnANode(client, node)
+		dpm := listDuplicatePodsOnANode(client, node)
```

Comment:
I have intentionally kept all exported for time being for faster development, as things will be changing fast.